echo-web/config/echo.yml

```diff
@@ -2,16 +2,24 @@ server:
   port: 8089
 
 spinnaker:
+  inMemory:
+    enabled: true
   cassandra:
     cluster: 'spinnaker'
     keyspace: 'echo'
-  baseUrl: 'http://localhost:9000'
+  baseUrl: ${services.deck.baseUrl:http://localhost:9000}
 
 front50:
-  baseUrl: 'http://localhost:8080'
+  enabled: ${services.front50.enabled:false}
+  baseUrl: ${services.front50.baseUrl:http://localhost:8080}
+
+igor:
+  enabled: ${services.igor.enabled:false}
+  baseUrl: ${services.igor.baseUrl:http://localhost:8088}
 
 orca:
-  baseUrl: 'http://localhost:8083'
+  enabled: ${services.orca.enabled:false}
```

Comment:
This will change the default, as currently the code defaults `orca.enabled` to `true`.  I think this should default to `true`.

Reply:
updating, odd that the configs had default to false, but it was defaulted to true in the code O_o